### PR TITLE
Rewrite the Getting Started page

### PR DESCRIPTION
**Pull request by tas50**
Bring getting started into 2021

Signed-off-by: Tim Smith <tsmith@chef.io>

https://deploy-preview-1740--chef-workstation.netlify.app/workstation/getting_started

### CODE REVIEW

File: docs-chef-io/content/workstation/getting_started.md [hunk 1]
```diff
@@ -19,60 +19,37 @@ new Chef Workstation installation. If you do not have Chef Workstation
...
+Chef Infra support in editors:
 
-3. Run `which ruby` again. It should return
-    `/opt/chef-workstation/embedded/bin/ruby`.
+- [VSCode Chef Infra Extension](https://marketplace.visualstudio.com/items?itemName=chef-software.Chef)
+- [Chef on Atom](https://atom.io/packages/language-chef)
+## Set up Ruby
 
-{{< note >}}
+While you don't have to know the Ruby programming language to use Chef, you do need to make sure that your computer knows where to find Ruby in order to run.
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:41]: 💯

File: docs-chef-io/content/workstation/getting_started.md [hunk 2]
```diff
@@ -19,60 +19,37 @@ new Chef Workstation installation. If you do not have Chef Workstation
...
-    ``` bash
-    echo 'eval "$(chef shell-init zsh)"' >> ~/.zshrc
-    ```
+Chef Infra support in editors:
 
-3. Run `which ruby` again. It should return
-    `/opt/chef-workstation/embedded/bin/ruby`.
+- [VSCode Chef Infra Extension](https://marketplace.visualstudio.com/items?itemName=chef-software.Chef)
+- [Chef on Atom](https://atom.io/packages/language-chef)
+## Set up Ruby
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:39]:
```suggestion

## Set up Ruby
```

File: docs-chef-io/content/workstation/getting_started.md [hunk 3]
```diff
@@ -19,60 +19,37 @@ new Chef Workstation installation. If you do not have Chef Workstation
...
+support the following: themes, plugins, snippets, syntax Ruby code
+coloring/highlighting, multiple cursors, a tree view of the entire
+folder/repository you are working with, and a Git integration.
 
-    where `bash` and `~/.bash_profile` represents the name of the shell.
+These are a few common editors:
 
-    If zsh is your preferred shell then run the following:
+- [Visual Studio Code (free/open source)](http://code.visualstudio.com)
+- [GitHub Atom - (free/open source)](http://atom.io)
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:33]: We probably want to remove mention of Atom at some point. The integration isn't maintained and Atom is slowly dying.

File: docs-chef-io/content/workstation/getting_started.md [hunk 4]
```diff
@@ -19,60 +19,37 @@ new Chef Workstation installation. If you do not have Chef Workstation
...
 
-{{< /note >}}
+### Add Ruby as an Environment Variable
 
-Add Ruby to \$PATH
-==================
+Chef Workstation includes a stable version of Ruby as part of its installer.
+The path to this version of Ruby must be added to the `$PATH` environment variable and saved in the configuration file for the command shell (Bash, csh, and so on) that is used on the machine running Chef Workstation.
 
-Chef Infra Client includes a stable version of Ruby as part of its
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:71]: I like how we talked about infra client here even though we're in workstation. 🤦

File: docs-chef-io/content/workstation/getting_started.md [hunk 5]
```diff
@@ -198,22 +196,19 @@ chef-server-ctl org-create ORG_NAME ORG_FULL_NAME -f FILE_NAME
...
 - `-f FILE_NAME`: Write the `ORGANIZATION-validator.pem` to
     `FILE_NAME` instead of printing it to `STDOUT`. For example:
-    `/tmp/chef.key`.
+    `/tmp/4cafe.key`.
 
-For example, an organization named `chef`, with a full name of
-`Chef Software, Inc.`, and with the ORGANIZATION-validator.pem file
-saved to `/tmp/chef.key`:
+For example, an organization named `4cafe`, with a full name of
+`Fourthe Cafe, Inc.`, and with the ORGANIZATION-validator.pem file
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:207]: This validator stuff is all wrong, but we need to fix that another time

File: docs-chef-io/content/workstation/getting_started.md [hunk 6]
```diff
@@ -19,60 +19,37 @@ new Chef Workstation installation. If you do not have Chef Workstation
...
+## Set up Ruby
 
-{{< note >}}
+While you don't have to know the Ruby programming language to use Chef, you do need to make sure that your computer knows where to find Ruby in order to run.
 
-Using Chef Workstation-provided Ruby as your system Ruby is optional.
-For many users, Ruby is primarily used for authoring Chef cookbooks. If
-that's true for you, then using the Chef Workstation-provided Ruby is
-recommended.
+* If you don't plan on writing a lot of Chef cookbooks and you don't use Ruby for other purposes, then set up Ruby with an [environment variable]({{< ref "#Add Ruby as an Environment Variable" >}})
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:43]: I don't think this is entirely correct. I like the structure and tone, but we need to get the reasons right:

- You just want chef to work and your not a ruby developer elsewhere: use our ruby
- You do ruby development with your system ruby: env vars
- You use RVM: let's not document that. It's all a mess of terribleness

**Review by kagarmoe — COMMENTED** [review]
There's a bit more work to do reorganizing the config section. I'll look at that in the next day or two.

The sections on ad-hoc chef run and installing a NTP server are probably good next steps, but they don't make sense in the setup doc.

1. Let's move the next step demo into a separate doc
2. Are these the best next steps? What do we want users to try out first?

**Comment by tas50** [comment]
We can probably skip the whole chef-run bit. I'm still a bit unsure how to merge chef-run with everything else chef in these docs, but it 100% doesn't need to be here. The best command to confirm everything works is probably `knife status`

File: docs-chef-io/content/workstation/getting_started.md [hunk 7]
```diff
@@ -140,7 +140,9 @@ On Microsoft Windows, `C:/opscode/Chef Workstation/bin` must be before
 
 {{% chef_repo_description %}}
 
-Use [the chef generate repo]({{< relref "ctl_chef.md#chef-generate-repo" >}}) command to create the Chef repository. For example, to create a repository called
+If you're setting up Chef for the very first time **in your organization**, then you will need a Chef repository for saving your cookbooks and other work.
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:143]: These should be Chef Infra. This stuff is all infra specific.

File: docs-chef-io/content/workstation/getting_started.md [hunk 8]
```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
...
+   1. `USER.pem`
+1. Install a [Code Editor]({{< relref "#Code Editors" >}}) (optional)
 
-1. Open a terminal and enter the following:
+## Setup Ruby for Chef Workstation
+
+While you don't have to know the Ruby programming language to use Chef, you do need to make sure that your computer knows where to find Ruby in order to run.
+
+* If you don't plan on writing a lot of Chef cookbooks and you're not a Ruby developer, then we recommend that you use the [Chef Workstation Ruby as the system default]({{< relref "#Use Chef-Workstation Ruby as the System Default">}})
+* If you are a Ruby developer, then set up Ruby with an [environment variable]({{< relref "#Add Ruby as an Environment Variable" >}}).
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:41]: `Add Ruby as an Environment Variable` doesn't really make sense as a way to describe this. It should be adding ruby to the system PATH

File: docs-chef-io/content/workstation/getting_started.md [hunk 9]
```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
...
+Chef Workstation installs two different things onto your computer:
 
-- `config.rb`. This configuration file can be downloaded from the **Organizations** page.
-- `ORGANIZATION-validator.pem`. This private key can be downloaded from the **Organizations** page.
-- `USER.pem`. This private key can be downloaded from the **Change Password** section of the **Account Management** page.
+1. [Chef products and tools]({{< relref "install_workstation.md" >}})
+1. The `~/.chef` directory
 
-#### Move Config Files
+The `~/.chef` directory starts out with one file, which is the `config.rb` with example configuration settings.
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:159]: This isn't actually the file we create. This setup is referencing the old config.rb setup. Here.s the new files created by the Workstation app:

```
❰tsmith❙~/.chef❱✔≻ ls
credentials
❰tsmith❙~/.chef❱✔≻ cat credentials
# This is the Chef Infra credentials file used by the knife CLI and other tools
# This file supports defining multiple credentials profiles, to allow you to switch between users, orgs, and Chef Infra Servers.

# Example credential file configuration:
# [default]
# client_name = 'MY_USERNAME'
# client_key = '/Users/tsmith/.chef/MY_USERNAME.pem'
# chef_server_url = 'https://api.chef.io/organizations/MY_ORG'
```

File: docs-chef-io/content/workstation/getting_started.md [hunk 10]
```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
...
-    ```
+Setting up Chef Workstation to trust the server's self-signed certificates you can use the `knife ssl fetch` subcommand to download the TLS/SSL certificate from the Chef Infra Server:
 
-    and:
+To set up Chef Workstation to communicate with your Chef Infra Server, you need the to download the following files in your `~.chef` directory:
 
-    ``` bash
-    cp /path/to/ORGANIZATION-validator.pem ~/chef-repo/.chef
-    ```
+* The ORGANIZATION-validator.pem
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:171]: We don't want folks to touch the validator anymore. They only need their user.pem file.

File: docs-chef-io/content/workstation/getting_started.md [hunk 11]
```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
...
 
-1. Open a terminal and enter the following:
+## Setup Ruby for Chef Workstation
+
+While you don't have to know the Ruby programming language to use Chef, you do need to make sure that your computer knows where to find Ruby in order to run.
+
+* If you don't plan on writing a lot of Chef cookbooks and you're not a Ruby developer, then we recommend that you use the [Chef Workstation Ruby as the system default]({{< relref "#Use Chef-Workstation Ruby as the System Default">}})
+* If you are a Ruby developer, then set up Ruby with an [environment variable]({{< relref "#Add Ruby as an Environment Variable" >}}).
+
+### Use Chef Workstation Ruby as the System Default ¸
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:43]:
```suggestion
### Use Chef Workstation Ruby as the System Default
```

File: docs-chef-io/content/workstation/getting_started.md [hunk 12]
```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
...
 
-Using Chef Workstation-provided Ruby as your system Ruby is optional.
-For many users, Ruby is primarily used for authoring Chef cookbooks. If
-that's true for you, then using the Chef Workstation-provided Ruby is
-recommended.
+   should return:
 
-{{< /note >}}
+   ``` bash
+   /opt/chef-workstation/embedded/bin/ruby`
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:88]:
```suggestion
   /opt/chef-workstation/embedded/bin/ruby
```

File: docs-chef-io/content/workstation/getting_started.md [hunk 13]
```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
...
-folder/repository you are working with, and a Git integration.
-
-These are a few common editors:
+{{< /foundation_tabs_panel >}}
+{{< foundation_tabs_panel panel-id="ruby-env-windows-panel" >}}
+{{< warning >}}
 
-- [Visual Studio Code (free/open source)](http://code.visualstudio.com)
-- [GitHub Atom - (free/open source)](http://atom.io)
+  On Microsoft Windows, `C:/opscode/Chef Workstation/bin` must be before `C:/opscode/Chef Workstation/embedded/bin` in the `PATH`.
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:132]: We probably want to describe how to add to the path via either the GUI or via PowerShell here before we warn them about the order.

File: docs-chef-io/content/workstation/getting_started.md [hunk 14]
```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
...
-If you have access to Chef Infra Server through Automate or Chef Manage,
-you can download the starter kit. The starter kit will create the
-necessary configuration files: the `.chef` directory, `config.rb`,
-`ORGANIZATION-validator.pem`, and `USER.pem`. Simply download the
-starter kit and then move it to the desired location on your Chef
-Workstation machine.
+{{% chef_repo_description %}}
 
-## Configure the Chef Repository
+If you're setting up Chef for the very first time **in your organization**, then you will need a Chef repository for saving your cookbooks and other work.
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:143]:
```suggestion
If you're setting up Chef Infra for the very first time **in your organization**, then you will need a Chef Infra repository for saving your cookbooks and other work.
```

File: docs-chef-io/content/workstation/getting_started.md [hunk 15]
```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
...
+chef generate repo chef-repo
+```
 
-#### Get Config Files
+## Your Chef Workstation Installation
 
-For a Chef Workstation installation that will interact with the Chef
-Infra Server (including the hosted Chef Infra Server), log on and
-download the following files:
+Chef Workstation installs two different things onto your computer:
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:154]: We probably need to reword this. Chef Workstation doesn't actually create the .chef dir. That happens the first time the Workstation App is launched. That's important as a CLI user will not get the directory.

File: docs-chef-io/content/workstation/getting_started.md [hunk 16]
```diff
@@ -318,93 +323,114 @@ See [SSL Certificates](/chef_client_security/#ssl-certificates) for
 more information about how knife and Chef Infra Client use SSL
 certificates generated by the Chef Infra Server.
 
-## Verify Server Communication
+### From Hosted Chef or Chef Manage
```
Comment (tas50) [inline docs-chef-io/content/workstation/getting_started.md:326]: The problem we're going to run into here is that the starter kit steps produce a config.rb and we want users to use the credentials file. This is a :( situation

File: docs-chef-io/content/workstation/getting_started.md (hunk 11 above)
Comment (kagarmoe) [reply docs-chef-io/content/workstation/getting_started.md:43]: I thought that was a burnt pixel on my screen!

**Review by kagarmoe — APPROVED** [review]
WOOT!

**Comment by tas50** [comment]
There's more to change here, but this is 90% accurate

**Pull request merged**